parm/stage/ocean_control_RT.yaml.j2

```
@@ -2,7 +2,7 @@ ocean_control_RT:
     mkdir:
         - "{{ COMOUT_OCEAN_RESTART_PREV_MEM }}"
         - "{{ COMOUT_OCEAN_ANALYSIS_MEM }}"
-    copy:
+    link:
```

```suggestion
    link_req:
```
Use `link_req` to ensure that it checks the file is in place.